lib/tasks/update_subscriptions.rake

```diff
@@ -0,0 +1,14 @@
+namespace :subscription_records do
+  desc 'converts old working pattern subscriptions to array style'
+  task convert_subscriptions: :environment do
+    Subscription.all.each do |s|
+      search_criteria_hash = s.search_criteria_to_h
+      search_criteria_hash.transform_keys! { |k| k.gsub(/working_pattern\b/, 'working_patterns') }
```

Comment:
Could you please include a comment explaining the old format of the keys and giving a quick example of how one would change?  The word boundary marker makes me realise I don't know what a valid and complete example of a key might look like.